packages/react-components/src/components/styles/Captions.style.ts

```diff
@@ -42,5 +42,6 @@ export const captionContainerClassName = mergeStyles({
  */
 export const captionsBannerClassName = mergeStyles({
   height: _pxToRem(100),
-  overflow: 'scroll'
+  overflowY: 'scroll',
```

Comment:
This shouldn't be scroll otherwise the scroll bar will always shows even there is only one caption

```suggestion
  overflowY: 'auto',
```